x/storage/keeper/keeper.go

```diff
@@ -334,6 +335,9 @@ func (k Keeper) ForceDeleteBucket(ctx sdk.Context, bucketId sdkmath.Uint, cap ui
 				ctx.Logger().Error("unlock store fee error", "err", err)
 				return false, deleted, err
 			}
+			if ctx.IsUpgraded(upgradetypes.Pawnee) {
```

Comment:
For this, I think the SP need run a job: scan all the stale created object and reject seal them.

Reply:
If we can reject seal the stale created object, it will be better. I checked greenfieldscan, cannot find the stale created objects easily.